frontend-react/src/components/MessageTracker/WarningsErrors.tsx

```diff
@@ -1,6 +1,5 @@
 import React from "react";
 
-import { WarningErrorDetail } from "../../config/endpoints/messageTracker";
```

Comment:
If we remove this the `WarningsErrorsDisplayProps` will break

Reply:
All of the API data types were moved to one file that is globally declared for the project and do not need to be imported.